src/TLinkStore.h

```diff
@@ -25,12 +25,14 @@
 #include <QStringList>
```

Comment:
GH Actions seems to be suggesting you need this here...

```suggestion
#include <QStringList>
#include <QVector>
```